src/java_bytecode/java_entry_point.cpp

```diff
@@ -400,16 +482,64 @@ bool java_entry_point(
     {
       message.error() << "main method `" << main_class
                       << "' has no body" << messaget::eom;
-      return true; // give up with error
+      res.main_function=symbol;
+      res.error_found=true;
+      res.stop_convert=true;
+      return res;  // give up with error
     }
   }
 
+  res.main_function=symbol;
+  res.error_found=false;
+  res.stop_convert=false;
+  return res;  // give up with error
+}
+
+/*******************************************************************\
+
+Function: java_entry_point
+
+  Inputs: symbol_table
+          main class
+          message_handler
+          allow pointers in initialization code to be null
+
+ Outputs: true if error occurred on entry point search
+
+ Purpose: find entry point and create initialization code for function
+
+\*******************************************************************/
+
+bool java_entry_point(
+  symbol_tablet &symbol_table,
+  const irep_idt &main_class,
+  message_handlert &message_handler,
+  bool assume_init_pointers_not_null,
+  size_t max_nondet_array_length)
```

Comment:
I added `false` and `0` as default values, is there any intention to have this as parameter for CEGIS or is this sufficient?